### DISCUSSION TITLE
Plain text doesn't use "Code mode" anymore

### DISCUSSION THREAD

**Discussion by poige** [coteditor/CotEditor#1643] [General]
I think it's changed a few updates ago; why that change and how to revert it back?

**Comment by 1024jp** [comment]
The syntax kind of Plain Text syntax is General from the beginning of the implementation of the syntax kind option, while the kind of None syntax is Code.
If you want to change the mode for Plain Text syntax, you have two choices:

## 1. Make a custom mode for Plain Text in the Mode settings

You can create your own mode for specific syntax. To do so, go to the Mode settings pane and add Plain Text from the plus button below the mode selector.
Then set the mode for Plain Text exactly like the Code mode.

<img width="712" alt="Screenshot 2024-05-23 at 18 46 08" src="https://github.com/coteditor/CotEditor/assets/1165044/cd6c9a40-93b1-4324-aeee-1103b1a9056f">


## 2. Change the syntax kind of Plain Text to Code

Go to the Format settings pane and edit Plain Text syntax by selecting the Plain Text syntax in the "Available Syntaxes" list and click the edit (pencil) button. In the syntax editor, you can select the kind of syntax in the popup button at the top right.
Once you save the setting, the Plain Text syntax will be handled as Code mode.

<img width="712" alt="Screenshot 2024-05-23 at 18 42 05" src="https://github.com/coteditor/CotEditor/assets/1165044/592adc6d-2c82-4bb0-97a7-1aef9409470c">

<img width="792" alt="Screenshot 2024-05-23 at 18 42 29" src="https://github.com/coteditor/CotEditor/assets/1165044/a656fff4-98b8-44b6-b4f2-305334362122">


I personally recommend the first option, but the result in this case is the same.

**Comment by poige** [comment]
Yeah, thank you for work-arounds suggested, but was there a change a few releases ago? — because I always used Coteditor as described, not sure if I really used any specific mode, but pretty sure it worked differently just not that long ago.

**Reply by 1024jp** [reply]
Yes. Actually, the concept of the mode itself has just been introduced recently on CotEditor 4.8.0, which was released this April. 

As a side note, CotEditor has been avoiding actively announcing new features to the user in the app until so far. However, I have recently changed my mind, and from the next minor update, I will announce such important changes in the so-called What's New panel.

**Reply by poige** [reply]
I see, thanks for explanation. I preferred using CotEditor because of its simplicity, to be honest. But these changes clearly kicked this editor out of the KISS-followers league.
